tests/sentry/middleware/test_access_log_middleware.py

```diff
@@ -183,7 +183,7 @@ def test_access_log_success(self):
         self.login_as(user=self.create_user())
         self.get_success_response(extra_headers={"HTTP_AUTHORIZATION": f"Bearer {token.token}"})
         self.assert_access_log_recorded()
-        assert self.captured_logs[0].token_type == "ApiToken"
+        assert self.captured_logs[0].token_type == "api_token"
```

Comment:
@Milkshak3s Just a heads up. It looks like the value for `token_type` in the logs query/use will be changing from `ApiToken` to `api_token`.

Reply:
@mdtro Yes -- as per this PR and moving forward, the original token class(es) get replaced with a single unified model `AuthenticatedToken` which maps back to this casing of the source of that token.  Is the casing important?  I could convert all of my output types to retain compatibility but ideally, the casing helps clarify this difference between storage types (`ApiToken`) and representation types (`AuthenticatedToken.token_type == 'api_token'`).  This is important because the backing storage can also differ based on silo (`ApiTokenReplica` for region silos)